ts/features/bonus/cgn/components/detail/CgnCardComponent.tsx

```diff
@@ -0,0 +1,190 @@
+import { playSvg } from "./CardSvgPayload";
+
+type Props = {
+  cgnDetails: Card;
+} & ReturnType<typeof mapStateToProps>;
+
+const styles = StyleSheet.create({
+  cardContainer: {
+    height: "100%"
+  },
+  cgnCard: {
+    position: "absolute",
+    width: "100%",
+    height: 192,
+    shadowColor: "#000",
+    shadowOffset: {
+      width: 0,
+      height: 7
+    },
+    shadowOpacity: 0.29,
+    shadowRadius: 4.65,
+    zIndex: 7,
+    elevation: 7,
+    top: 2
+  },
+  informationContainer: {
+    height: "100%",
+    top: -190,
+    zIndex: 9,
+    elevation: 9
+  },
+  spaced: {
+    justifyContent: "space-between"
+  },
+  flex1: {
+    flex: 1
+  },
+  flex2: {
+    flex: 2
+  },
+  paddedContentFull: {
+    paddingLeft: 16,
+    paddingTop: 24,
+    paddingRight: 20,
+    paddingBottom: 16
+  },
+  column: {
+    flexDirection: "column"
+  },
+  fullLogo: {
+    resizeMode: "contain",
+    height: 56,
+    width: 56,
+    alignSelf: "flex-end"
+  },
+  imageFull: {
+    resizeMode: "stretch",
+    height: "100%",
+    zIndex: 8,
+    elevation: 8
+  }
+});
+
+const minPointA: Point = {
+  x: 80,
+  y: -100
+};
+const maxPointA: Point = {
+  x: 100,
+  y: 50
+};
+
+const minPointB: Point = {
+  x: -80,
+  y: 0
+};
+const maxPointB: Point = {
+  x: 100,
+  y: 10
+};
+
+const minPointC: Point = {
+  x: -50,
+  y: 50
+};
+const maxPointC: Point = {
+  x: 50,
+  y: 10
+};
+
+const MOVEMENT_STEPS = 12;
+
+const CgnCardComponent: React.FunctionComponent<Props> = (props: Props) => {
+  const generatedTranslationA = generateRandomSvgMovement(
+    MOVEMENT_STEPS,
+    minPointA,
+    maxPointA
+  );
+  const generatedTranslationB = generateRandomSvgMovement(
+    MOVEMENT_STEPS,
+    minPointB,
+    maxPointB
+  );
+  const generatedTranslationC = generateRandomSvgMovement(
+    MOVEMENT_STEPS,
+    minPointC,
+    maxPointC
+  );
+
+  const generatedSvg = playSvg(
+    generatedTranslationA,
+    generatedTranslationB,
+    generatedTranslationC
+  );
+
+  return (
+    <View style={[IOStyles.horizontalContentPadding, styles.cgnCard]}>
+      <ImageBackground
+        source={cardBg}
+        imageStyle={[styles.imageFull, { borderRadius: 10 }]}
+        style={styles.cardContainer}
+      >
+        <WebView
+          style={{ borderRadius: 15, top: -1, left: -1 }}
+          source={{
+            html: generatedSvg
+          }}
+        />
+      </ImageBackground>
+      <View style={[styles.informationContainer, styles.paddedContentFull]}>
+        <View
+          style={[
+            IOStyles.row,
+            IOStyles.flex,
+            { justifyContent: "space-between" }
+          ]}
+        >
+          <View style={[styles.column, styles.flex2, styles.spaced]}>
+            <H3 weight={"Bold"} color={"black"}>
+              {I18n.t("bonus.cgn.name")}
+            </H3>
+            <View>
+              {props.cgnDetails.status !== "PENDING" && (
+                <H5>
+                  {"Valida fino al " +
```

Comment:
```suggestion
                  {"Valida fino al " +
```
this should be translated at runtime

Reply:
Missed it, ty!